mne/io/tests/test_raw.py

```diff
@@ -582,3 +582,61 @@ def test_describe_df():
                               -9.643554807784935e-13,
                               1.928710961556987e-12,
                               4.146728567347522e-11]))
+
+
+def test_get_data_units():
```

Comment:
I would suggest to put the conversion logic in a dedicated private function that you can test with a test that is very fast to run.

I would do:

```
# put here a pytest parametrize.
def test_convert_unit(orig_unit, dest_unit, factor):
     factor_ = _get_unit_convert_factor(orig_unit, dest_unit)
     assert factor == factor_
```

doing this will make it obvious what is tested and if it's correct.

then you can have one test that just calls get_data